stage1_fly/run/main.go

```diff
@@ -333,8 +338,8 @@ func stage1() int {
 		return 1
 	}
 
-	if err := os.Chdir("/"); err != nil {
-		log.PrintE("can't change to root new directory", err)
+	if err := os.Chdir(workDir); err != nil {
```

Comment:
The `chroot()` call happens a couple lines above so the `Chdir()` is already inside the new RFS. Would `../` have a negative effect at all?